sitestats-impl/src/bundle/site-stats-components.xml

```diff
@@ -233,14 +232,14 @@
 	</bean>
 	<!-- Transaction proxy: StatsUpdateManager -->
 	<bean id="org.sakaiproject.sitestats.api.StatsUpdateManagerTransactionProxyFactoryBean"
-		class="org.springframework.transaction.interceptor.TransactionProxyFactoryBean">		
+		class="org.springframework.transaction.interceptor.TransactionProxyFactoryBean">
 		<property name="transactionManager" ref="@{sitestats.db}SitestatsTransactionManager" />
 		<property name="target" ref="org.sakaiproject.sitestats.api.StatsUpdateManager"/>
 		<property name="transactionAttributes">
 			<props><prop key="*">PROPAGATION_REQUIRED</prop></props>
 		</property>
 	</bean>
-	
+
 

```

Comment:
Good spot, normally I'd fix this but this is already in Sakai as is so it's not to much of a problem having it locally.